src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -456,17 +483,21 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
               </Button>
             </>
           )}
-          {artwork.is_inquireable &&
-            !artwork.is_acquireable &&
-            !artwork.is_offerable && (
-              <Button
-                width="100%"
-                size="medium"
-                onClick={this.handleInquiry.bind(this)}
-              >
-                Contact Gallery
-              </Button>
-            )}
+          {shouldDisplayContactGalleryButton() && (
+            <Button
+              width="100%"
+              size="medium"
+              onClick={this.handleInquiry.bind(this)}
+              mt={shouldDisplayMakeOfferAsPrimary() ? 2 : 0}
```

Comment:
Just curious: I'm still not sure when we use `mt` on a Spacer vs. on the component itself. For consistency sake, should we use `mt` also on the Make offer button, or use a Spacer here, or is it ok to have both?